CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Submission Data API
 
+### v6.8.0
+* Remove products marked with showcondition in modify submission additional products command
```

Comment:
I'd change the comment to something like
* remove obsolete submission additional products